submissions/Oleksandr-Heleta/OOP_exercise/index.js

```diff
@@ -0,0 +1,108 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitants {
+   constructor(species, name, gender, legs, say) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.say = say;
+   }
+
+   createString() {
+      return `${this.species}, ${this.name}, ${this.gender}, ${this.legs}, ${this.say};`;
+   }
+}
+
+class Human extends Inhabitants {
+   constructor(name, gender, say) {
+      super('human', name, gender, 2, say);
+      this.hands = 2;
+   }
+
+   createString() {
+      return `${this.species}, ${this.name}, ${this.gender}, ${this.legs}, ${this.hands}, ${this.say};`;
+   }
```

Comment:
You did method overloading but you do not benefit from explicit inherited method version call.
This wil make code dryer and hence less error prone.
Check learning materials.
Read comments in their entirety.
Ask questions if anything is not clear.